lib/player.js

```diff
@@ -2074,6 +2053,37 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     });
   }
 
+  /**
+   * @param {!Event} event
+   * @private
+   */
+  onAddTextTrack_(event) {
+    this.onTracksChanged_();
+    const track = event.track;
```

Comment:
nit: add a comment that we're looking for metadata tracks to process id3 tags for ad info on LIVE streams

Reply:
Added!